### PR TITLE
[Website 2.0] Artifact URL Adjustment

### PR DESCRIPTION
**Pull request by connorgoggins**
## Description ##
This PR moves static artifacts to MXNet website artifacts public bucket.

### Essentials ###
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] Code is well-documented
- [x] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- M docs/static_site/Makefile

## Comments ##
These changes have been tested with a complete website build on Jenkins dev and local hosting. All functionality remains the same.

@sandeep-krishnamurthy @aaronmarkham @ys2843

### CODE REVIEW

**Comment by connorgoggins** [comment]
@mxnet-label-bot add [pr-awaiting-review]

**Comment by connorgoggins** [comment]
@mxnet-label-bot add [Website]

File: docs/static_site/Makefile [hunk 1]
```diff
@@ -20,7 +20,7 @@ all: html
 html:
 	mkdir -p build
 	cd src && bundle install && JEKYLL_ENV=production bundle exec jekyll build --config _config_prod.yml -d ../build/html && cd ..
-	wget https://mxnet-static-artifacts.s3.amazonaws.com/versions.zip && unzip versions.zip -d build/html && rm -rf build/html/__MACOSX && rm versions.zip
+	wget https://mxnet-website-static-artifacts.s3.us-east-2.amazonaws.com/versions.zip && unzip versions.zip -d build/html && rm -rf build/html/__MACOSX && rm versions.zip
```
Comment (aaronmarkham) [inline docs/static_site/Makefile:23]: What's the macosx folder for?

Comment (connorgoggins) [reply docs/static_site/Makefile:23]: It gets autogenerated when unpacking the zip, since the archive was created on a Mac.

**Pull request merged**